content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,154 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+### Attaching Files
+
+Attaching files to input elements or dropping them over the page is available
+in Cypress 10.0. Read the [`.attachFile()` API docs](/api/commands/attachfile)
+for more information on how this works and how to use it.
+
+The
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload)
+plugin has been deprecated in favor of `.attachFile()` built into Cypress.
+There's guidance below on how to migrate from the
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload)
+plugin to Cypress's built-in attachFile command.
+
+#### Quick guide
+
+The argument signature is different for `.attachFile()` than the `cypress-file-upload` plugin. You can follow the steps below for each argument in order to migrate:
```

Comment:
This is technically correct. I've only learned this recently because I have kids but they hate when people correct them on it 🤣 

```suggestion
The argument signature is different for `.attachFile()` from the `cypress-file-upload` plugin. You can follow the steps below for each argument in order to migrate:
```

Reply:
It's not different for attachFiles() *from* cypress-file-upload though, that would mean something different!

I'll tweak the working a bit.